test/spec/upload.js

```diff
@@ -81,15 +83,21 @@ describe("tus", function () {
         withCredentials: true,
         urlStorage: getStorage(),
         onProgress: function () {},
-        fingerprint: function () {}
+        fingerprint: function (_file, _options, cb) {
+          passedOptions = _options;
```

Comment:
I think it should be enough to use `expect(_file).toBe(file)` and the same for `options` here. Or is there a particular reason for why you didn't do it?

Reply:
yeah, I just wanted it to be clear at what point the assertion is actually required.
If it is added into the fingerprint function as you suggest, it may mislead one to think that it is called at that instant. Or it even just makes one unsure when the assertion would be triggered.

Reply:
I don't agree with you here but whatever. Let's not get hung up on this, we got more important stuff to do ;)